### PR TITLE
Gripperfix master

### PR DESCRIPTION
**Pull request by MatthijsBurgh**
As the robocup branch is not merged, and shouldn't be merged anymore

### CODE REVIEW

**Comment by MatthijsBurgh** [comment]
@LarsJanssenTUe Please only check for typo's etc.

File: parameters/hardware/actuators/left_arm_controller.ops [hunk 1]
```diff
@@ -76,6 +76,8 @@ EtherCATread.left_arm1_matrixtransform8 = array( 0.0,  0.0,  0.0,  0.0,  0.0,  0
 EtherCATread.AddSaturation_E(BODYPARTNAME,1, LOWERJOINTLIMIT, UPPERJOINTLIMIT)
 EtherCATread.AddMsgOut_E(BODYPARTNAME, 1,JOINT_NAMES);
 EtherCATread.AddTorqueSensor_A(BODYPARTNAME, 1, SENSORTORQUES_C1, SENSORTORQUES_C2, SENSORTORQUES_C3)
+#EtherCATread.AddMsgOut_A(BODYPARTNAME,1); # Can be used for force sensor tuning
```
Comment (LarsJanssenTUe) [inline parameters/hardware/actuators/left_arm_controller.ops:79]: Maybe state also how this can be done?

**Review by LarsJanssenTUe — APPROVED** [review]
Just a bunch of (magic) numbers are change to values that seem reasonable.

File: parameters/hardware/actuators/left_arm_controller.ops (hunk 1 above)
Comment (MatthijsBurgh) [reply parameters/hardware/actuators/left_arm_controller.ops:79]: @MaxBaeten

**Pull request merged**